Sources/RoutingKit/Parameters.swift

```diff
@@ -11,13 +12,25 @@ public struct Parameters {
     /// Internal storage.
     private var values: [String: String]
     private var catchall: Catchall
+    private var logger: Logger
```

Comment:
This should be a `public let` so we can access it from extensions (like we need to in Vapor)

Reply:
done